9.21.0 RN

Adam-Dupaski opened the pull request.

Commit:
Add RN

Commit:
Remove duplicate notes

Commit:
Remove duplicate notes

Commit:
Remove duplicate notes

Commit:
Remove duplicate notes

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Add KI fix

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Add  via Slack

Commit:
Add via Slack

Commit:
Edit RN

Commit:
Edit RN

ConnorLand committed:
Add Fabians Page Editor Fix

Related to performance

ConnorLand commented on content/en/docs/releasenotes/studio-pro/9/9.21.md:
```diff
@@ -0,0 +1,87 @@
+* We fixed an issue where a thumbnail was not deleted when the image associated with the thumbnail was deleted. (Ticket 171367)
+* We fixed an issue in published OData services where resources with keys other than `id` threw an error when the client left out the key from the `$select` clause. (Ticket 171726)
+* We fixed an issue that occurred when editing a SCSS file and closing the editor with unsaved changes. (Ticket 171763)
+* We fixed an issue where the editors for pages, snippets, layouts, building blocks, and page templates would become very slow after copying a widget.
```
This fix addresses the issue in this [forum post](https://forum.mendix.com/link/space/studio%20pro/questions/120848).

Our fix was not directly inspired by the post, so I don't think we need to add an info box here saying "inspired by X".

Adam-Dupaski replied:
"would become" > "became"

ConnorLand replied:
Changed and resolved.

ConnorLand committed:
Change word

ConnorLand committed:
Add MPP anchor link

ConnorLand committed:
Add snippet parameter link

This anchor will work after this 9.21 PR is merged

https://github.com/mendix/docs/pull/5403

ConnorLand committed:
Change anchor

ConnorLand committed:
Add anchor link for widget style copy pasting

Commit:
Merge branch 'development' into ad-9.21.0-rn

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Add

MariaShaposhnikova committed:
Add links
